llama.cpp

```diff
@@ -385,13 +457,13 @@ static bool llama_model_load(
     fin.rdbuf()->pubsetbuf(f_buf.data(), f_buf.size());
 
     fin.seekg(0, fin.end);
-    const size_t file_size = fin.tellg();
+    const int64_t file_size = fin.tellg();
```

Comment:
check the file_size. we should have a very precise file size for different models.